scripts/language_model/data/transform.py

```diff
@@ -0,0 +1,258 @@
+"""XLNet dataset transform."""
+__all__ = ['XLNetSentenceTransform', 'XLNetDatasetTransform']
+import numpy as np
+
```

Comment:
No need for many blank lines. https://www.python.org/dev/peps/pep-0008/#blank-lines